packages/babel-traverse/src/scope/index.js

```diff
@@ -447,13 +447,17 @@ export default class Scope {
    * Possibly generate a memoised identifier if it is not static and has consequences.
    */
 
-  maybeGenerateMemoised(node: Object, dontPush?: boolean): ?Object {
+  maybeGenerateMemoised(
+    node: Object,
+    dontPush?: boolean,
+    getInjectionPath?: NodePath => void,
+  ): ?Object {
```

Comment:
I don't like this API, but it's the best I could come up with in a non-breaking way :confused:

Reply:
Me too. Can we not merge this to v8? Since we can have a cleaner fix without maintaining a confusing API (`scope.maybeGenerateMemoised` is available in plugins).

Reply:
It's easier to revert this PR in the v8 branch, so that v8 can still be "master + some commits"